### DISCUSSION TITLE
Retrieve values from 2 select fields

### DISCUSSION THREAD

**Discussion by rdmngithub** [livewire/livewire#4964] [1. Help]
I am trying to retrieve the values of 2 select fields.
With the following codes, I can't get 'a2b2':
```
<?php
namespace App\Http\Livewire;
use Livewire\Component;

class Test extends Component
{
    public $selectOne = 'a1';
    public $selectTwo = 'b1';

    public function updated($name, $value) {
        dd($this->selectOne . $this->selectTwo);
    }

    public function render()
    {
        return view('livewire.test');
    }
}
```

```
<div>
    <form>
        <select wire:model="selectOne" id="az">
            <option value="a1">a1</option>
            <option value="a2">a2</option>
        </select>

        <select wire:model="selectTwo" id="fd">
            <option value="b1">b1</option>
            <option value="b2">b2</option>
        </select>
    </form>
</div>
```
How to do ?
Thanks to those who will respond.

**Comment by NBA707** [comment]
For me, when I used updated, I did something like:
if ($name == 'selectOne') { 
 do something with $value; 
} elseif ($name == 'selectTwo') { 
 do something with $value; 
}

**Comment by joshhanley** [comment]
@rdmngithub Livewire and Select2 don't play nicely together out of the box. You will need to use Alpine or vanilla javascript to sit between the two to pass data back and forth. Have a search through the old issues and discussions on this repo as there are some threads on how to get Select2 running.

Hope this helps!
